sklearn/cluster/hierarchical.py

```diff
@@ -902,6 +902,16 @@ class FeatureAgglomeration(AgglomerativeClustering, AgglomerationTransform):
         node and has children `children_[i - n_features]`. Alternatively
         at the i-th iteration, children[i][0] and children[i][1]
         are merged to form node `n_features + i`
+
+    Examples
+    --------
+    >>> from sklearn.cluster import FeatureAgglomeration
+    >>> import numpy as np
+    >>> X = np.array([[1, 5], [2, 4], [2, 4],
```

Comment:
Feature agglomeration is different. It clusters features, not samples, and its use as a transformer needs to be illustrated

Reply:
example updated.